libkbfs/ops.go

```diff
@@ -867,10 +867,7 @@ func addToCollapsedWriteRange(writes []WriteRange,
 			// Truncate past the last write.
 			return append(head, wNew)
 		} else if mid[0].isTruncate() {
-			// Min truncate wins
-			if mid[0].Off < wNew.Off {
-				return append(head, mid[0])
-			}
+			// Later truncates always win.
```

Comment:
After this change, if a 3MB file is truncated to 1MB, then truncated to 2MB, does it retain the data from 1MB to 2MB or fill with `\0`?